storage/client/src/main/java/com/azure/storage/blob/AccountSASSignatureValues.java

```diff
@@ -186,8 +188,8 @@ public AccountSASSignatureValues withResourceTypes(String resourceTypes) {
      *
      * @return {@link SASQueryParameters}
      */
-    public SASQueryParameters generateSASQueryParameters(SharedKeyCredentials sharedKeyCredentials) {
-        Utility.assertNotNull("SharedKeyCredentials", sharedKeyCredentials);
+    public SASQueryParameters generateSASQueryParameters(SharedKeyCredential sharedKeyCredentials) {
+        Utility.assertNotNull("SharedKeyCredential", sharedKeyCredentials);
```

Comment:
Post preview we should look to changing these Utility.assertNotNull to Objects.requireNonNull

Reply:
Should we open an issue for this?

Reply:
I think this should be opened as an issue and cleaned up in another preview as this won't change any public interface or functionality.

See #4077